test/test_framers.py

```diff
@@ -64,47 +64,78 @@ def test_decode_data(rtu_framer, data):
     assert decoded == expected
 
 
-@pytest.mark.parametrize("data", [(b'', False),
-                                  (b'\x02\x01\x01\x00Q\xcc', True)])
+@pytest.mark.parametrize("data", [
+    (b'', False),
+    (b'\x02\x01\x01\x00Q\xcc', True),
+    (b'\x11\x03\x06\xAE\x41\x56\x52\x43\x40\x49\xAD', True),  # valid frame
+    (b'\x11\x03\x06\xAE\x41\x56\x52\x43\x40\x49\xAC', False),  # invalid frame CRC
+])
 def test_check_frame(rtu_framer, data):
     data, expected = data
     rtu_framer._buffer = data
     assert expected == rtu_framer.checkFrame()
 
 
-@pytest.mark.parametrize("data", [b'', b'abcd'])
-def test_advance_framer(rtu_framer, data):
-    rtu_framer._buffer = data
+@pytest.mark.parametrize("data", [
```

Comment:
Useless tests (which do not test anything but only improve code coverage) are replaced with new tests.